alfalfa_web/server/schema.js

```diff
@@ -149,6 +149,33 @@ const simType = new GraphQLObjectType({
   })
 });
 
+const runType = new GraphQLObjectType({
```

Comment:
Yes we are. But right now it's the only way to get information to the client. I also figured this could have give Alex some reference implementation for when he is moving it to a REST api.